TACC/TACC-Docs
# feat(css): migrate tup-cms.css to tacc/core-styles

## Overview

Migrate [TACC CMS](https://github.com/TACC/tup-ui/tree/main/apps/tup-cms) styles to Core-CMS.

## Related

- requires https://github.com/TACC/Core-Styles/pull/150
- similar to https://github.com/TACC/Core-CMS/pull/616
- similar to https://github.com/TACC/Core-CMS/pull/620

## Changes

- **changed** styles from `tup-cms.css` to be loaded from `core-styles.docs.css`
- **changed** to latest core-styles via https://github.com/TACC/Core-Styles/pull/150

## Testing

1. Verify figures have a border around the image.
2. Verify figure captions match CMS styles.

## UI

| docs | cms |
| - | - |
| ![figure](https://user-images.githubusercontent.com/62723358/229868615-b23f33fa-9658-48c2-8a80-0c78548ba9ee.png) | ![cms figure](https://user-images.githubusercontent.com/62723358/229868789-a0f7a195-5c90-4cb8-a7f9-0524eca2aab1.png) |
